vernac/attributes.mli

```diff
@@ -10,7 +10,7 @@
 
 (** The type of parsing attribute data *)
 type vernac_flag_type =
-  | FlagIdent of string
+  | FlagPath of Libnames.full_path
```

Comment:
This is a parser output, should be qualid not full_path

Reply:
done